tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+###
+### Main entrypoint
+###
+
+- include: install-helm.yml
+  tags:
+    # Leave this tag at the bottom
+    # Tests rely on its position
+    - helm-install
+
+- name: Overwrite Helm with the mock script
```

Comment:
My system does not have a `/etc/ansible/roles/rolename/tests/support/helm.mock` file. I suppose it will fail then at this stage?

Reply:
It should not since the test is running inside Docker. See `Makefile`

Reply:
Good catch! this was commited by accident. Removed.